docs/Install.md

```diff
@@ -7,84 +7,79 @@
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
 ```
 
+## Default Installation
 
-## Installation using a non-admin user
-
-Kubernetes only allows administrators to install new `CustomResourceDefinition` objects. All other objects included in the `splunk-operator.yaml` file can be installed by regular users within their own namespaces. 
-
-If you are not an administrator, you can have an administrator create the required objects for you by running:
-
-```
-kubectl apply -f https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-crds.yaml
-```
-
-Afterwards, you can download and use the yaml file to install the operator within your own namespace:
+By default operator will be installed in `splunk-operator` and will watch all the namespaces of your cluster for splunk enterprise custom resources:
 
 ```
-wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-noadmin.yaml
-kubectl config set-context --current --namespace=<NAMESPACE>
-kubectl apply -f splunk-operator.yaml
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
+kubectl apply -f splunk-operator-cluster.yaml
 ```
 
+## Install operator to watch single namespace
 
-## Admin Installation for All Namespaces
-
-If you want to configure a single instance of the operator to watch all the namespaces of your cluster while managing deployments only in the namespace `splunk-operator`, use the alternative cluster scope installation yaml file:
+By default operator will be installed in `splunk-operator` namespace and will watch all the namespaces of your cluster for splunk enterprise custom resources:
+if user wants to watch only one namespace then eeit `config-map` `splunk-operator-config` in `splunk-operaor` namespace, set `WATCH_NAMESPACE` field to the namespace operator should watch
 
 ```
-wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
+apiVersion: v1
+data:
+  OPERATOR_NAME: '"splunk-operator"'
+  RELATED_IMAGE_SPLUNK_ENTERPRISE: splunk/splunk:latest
+  WATCH_NAMESPACE: "namespace1"
+kind: ConfigMap
+metadata:
+  labels:
+    name: splunk-operator
+  name: splunk-operator-config
+  namespace: splunk-operator
 ```
 
-**Note**: The operator has the ability to watch, list secret objects in all namespaces.
-
-When running at cluster scope, you will need to bind the `splunk:operator:namespace-manager` ClusterRole to the `splunk-operator` ServiceAccount in all namespaces you want it to manage. 
+## Install operator to watch multiple namespace
 
-For example, to create a new namespace called `splunk` that is managed by Splunk Operator:
+if user want to manager multiple namespaces, then they can set `WATCH_NAMESPACE` field to all those namespaces , names should be comma (,) sepearated. example
```

Comment:
change line 43 to "If user wants to manage multiple namespaces, they must add the namespaces to the `WATCH_NAMESPACE` field, with each namespace separated by a comma (,). example

Reply:
Fixed